src/foundry/client/core/utils.d.ts

```diff
@@ -1,3 +1,5 @@
+import { ClientDocumentMixin } from '../data/abstract/client-document';
```

Comment:
Can you please check whether the functions in this file are available globally in foundry? If they are, you just removed that from the types by turning this file into an ES Module. If they are available globally, they need to be wrapped in `namespace global {}`.

Reply:
Fair catch, added a simple test to check it and wrapped the contents in `declare global` to fix said test.